client/src/components/CredentialsInputBox/CredentialsInputBox.module.css

```diff
@@ -0,0 +1,24 @@
+.credLayout {
```

Comment:
Same comment as the last comment regarding the `CredentialsInputBox.js` file.

It looks like the old code got merged in here and wasn't removed so we can remove the CredentialsInputBox folder.
The code has been moved in the InputBox and CredentialsForm folders

Reply:
Removed from current branch